### ISSUE TITLE
Provide as docker image

### ISSUE THREAD

**Issue by Malspherus** [mguessan/davmail#201]
It would be great if this project had an official docker image as there are already some fan-made ones available.

Some I found:
https://hub.docker.com/r/rsrchboy/davmail-savvis-docker/
https://hub.docker.com/r/connectical/davmail
https://hub.docker.com/r/kayvan/davmail
https://github.com/jberrenberg/docker-davmail
https://hub.docker.com/r/timonier/davmail
https://github.com/sebgru/docker-davmail

Advantages would include easier setup, faster releases (compared to the fan-made ones) and less worries about security. As there already is a linux version it shouldn't take a lot of work either

**Comment by UlyssesZh** [comment]
I would really like to see this. @mguessan Will this feature come eventually, or is it not planned?

**Comment by SethRobertson** [comment]
I'd like this as well.  I tried a number of the ones you mentioned and at the ones I tried didn't support O365 Interactive authentication (requiring a web browser like thing) and many were ancient.

I also needed the ability to compile, so I created a staged docker file that supported both.

I'm going to submit a pull request for this, but hopefully some Java clued person could fix it so that the CLASSPATH doesn't have to be hardcoded like I ended up doing.

**Comment by SethRobertson** [comment]
See pull request #381

**Comment by mguessan** [comment]
Thanks for your contribution, do I get correctly that the version you provide is docker based but interactive?

Mostly used Docker only in server headless mode so far

**Comment by SethRobertson** [comment]
Yes--I need interactive so that I can do third-party two factor authentication.

However, I'm sure you/we/someone could tweak it to not pop up the window, possibly just by changing the docker-start arguments, if a simple password (or OOB authentication) is all that is required.

**Comment by mguessan** [comment]
Reviewing old open tickets, thanks to user contributions we now provide a DavMail docket image published on GHCR at https://github.com/mguessan/davmail/pkgs/container/davmail
